x-pack/plugins/siem/public/components/page/hosts/uncommon_process_table/index.tsx

```diff
@@ -5,22 +5,18 @@
  */
 
 import { EuiBadge } from '@elastic/eui';
-import { get } from 'lodash/fp';
 import React from 'react';
 import { connect } from 'react-redux';
 import { pure } from 'recompose';
 import { ActionCreator } from 'typescript-fsa';
 
-import { HostEcsFields, UncommonProcessesEdges } from '../../../../graphql/types';
+import { UncommonProcessesEdges, UncommonProcessItem } from '../../../../graphql/types';
 import { hostsActions, hostsModel, hostsSelectors, State } from '../../../../store';
-import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
-import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
-import { defaultToEmptyTag, getEmptyTagValue, getEmptyValue } from '../../../empty_value';
-import { HostDetailsLink } from '../../../links';
+import { defaultToEmptyTag, getEmptyValue } from '../../../empty_value';
 import { Columns, ItemsPerRow, LoadMoreTable } from '../../../load_more_table';
-import { Provider } from '../../../timeline/data_providers/provider';
 
 import * as i18n from './translations';
+import { getRowItemDraggables } from '../../../tables/helpers';
```

Comment:
Really wish we still had import grouping from [ts-lint's ordered-imports](https://palantir.github.io/tslint/rules/ordered-imports/), but as from when we were doing the merge, doesn't look like the [es-lint equivalent](https://eslint.org/docs/rules/sort-imports) offers that configuration... 😞